src/pages/[platform]/build-a-backend/data/custom-business-logic/search-and-aggregate-queries/index.mdx

```diff
@@ -0,0 +1,431 @@
+
+Regarding the sink configuration, the OpenSearch domain cluster is specified by setting the host, index name, type, and template content (index mapping) for data formatting. Document-related metadata is configured along with the maximum bulk size for requests to OpenSearch in MB. Once again, an IAM role is specified for the sink portion of the pipeline. For further details on Sink configuration, please refer to the [OpenSearch documentation](https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/sinks/sinks/).
+
+The sink configuration is an array. To create a different index on the same table, you can achieve this by adding a second OpenSearch configuration to the sink array.
+
+"To index multiple tables, you'll need to configure multiple pipelines in the configuration. For further guidance, please consult the [pipeline section](https://opensearch.org/docs/latest/data-prepper/pipelines/pipelines/) of the OpenSearch documentation."
+
+
+NOTE: An OpenSearch Ingestion pipeline supports only one DynamoDB table as its source. For more details on current limitations, Please refer to [Amazon OpenSearch Limitation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/configure-client-ddb.html#ddb-pipeline-limitations) section.
+
+Now, create the OSIS pipeline resource:
+
+
+``` ts title="amplify/backend.ts"
+const logGroup = new logs.LogGroup(openSearchStack, "LogGroup", {
+  logGroupName: "/aws/vended-logs/OpenSearchService/pipelines/1",
+  removalPolicy: RemovalPolicy.DESTROY,
+});
+
+const cfnPipeline = new osis.CfnPipeline(
+  openSearchStack,
+  "OpenSearchIntegrationPipeline",
+  {
+    maxUnits: 4,
+    minUnits: 1,
+    pipelineConfigurationBody: openSearchTemplate,
+    pipelineName: "dynamodb-integration-2",
+    logPublishingOptions: {
+      isLoggingEnabled: true,
+      cloudWatchLogDestination: {
+        logGroup: logGroup.logGroupName,
+      },
+    },
+  }
+);
+
+```
+
+After deploying the resources, you can test the data ingestion process by adding an item to the `Todo` table. However, before doing that, let's verify that the pipeline has been set up correctly.
+
+In the AWS console, navigate to OpenSearch and then to the pipelines section. You should find your configured pipeline and review its settings to ensure they match your expectations:
+
+![A screenshot of OpenSearch OSIS pipeline](/images/gen2/opensearch-integration/OpenSearch_pipeline.png)
+
+You can also check this in the DynamoDB console by going to the Integrations section of the tables.
+
+![A screenshot of DynamoDB integration](/images/gen2/opensearch-integration/OpenSearch_DynamoDB_integration.png)
```

Comment:
Could we please get a more descriptive alt text for this that's explaining where we are in the UI?

Reply:
updated alt text.